osu.Game.Rulesets.Osu/Objects/Drawables/DrawableOsuHitObject.cs

```diff
@@ -51,6 +52,19 @@ protected virtual void UpdatePreemptState()
         protected virtual void UpdateCurrentState(ArmedState state)
         {
         }
+
+        private readonly WeakReference<OsuInputManager> osuActionInputManager = new WeakReference<OsuInputManager>(null);

```

Comment:
No I didn't, but I'm not sure that would work properly with the weak reference.  I guess I could try it?